codegen/protocol-test-codegen-local/Tests/swift-codegen/Tests/WaitersTests/WaiterTestUtils.swift

```diff
@@ -10,8 +10,11 @@ import Waiters
 import SmithyWaitersAPI
 
 // Convenience test-helper methods for testing acceptor matches
-
-extension WaiterConfiguration.Acceptor.Match: Equatable where Input: Equatable, Output: Equatable {
+//
+// Use of fully-qualified type names in this extension suppresses the Swift 6
+// "retroactive conformance" warning in a manner compatible with Swift 5.
+// See: https://github.com/swiftlang/swift-evolution/blob/main/proposals/0364-retroactive-conformance-warning.md#source-compatibility
+extension SmithyWaitersAPI.WaiterConfiguration.Acceptor.Match: Swift.Equatable where Input: Swift.Equatable, Output: Swift.Equatable {
```

Comment:
Using fully-qualified names in an extension declaration suppresses "retroactive conformance" warnings while still being compatible with Swift 5.x (using `@retroactive` in Swift 5 results in a compile error.)